docs/guides/getting-started.md

```diff
@@ -0,0 +1,120 @@
+---
+page_title: "Getting Started with the Prefect Cloud provider"
+description: |-
+  This guide walks through the necessary provider configurations,
+  how to generate a Prefect API Key, as well as a brief introduction
+  to Prefect RBAC and permissions
+---
+
+# Getting Started with the Prefect Cloud provider
+
+## Provider setup
+
+By default, the provider points to Prefect Cloud. You simply need to pass in an API Key and your Prefect Cloud Account ID.
+
+```terraform
+# provider.tf / versions.tf
+
+terraform {
+  required_providers {
+    prefect = {
+      source = "prefecthq/prefect"
+    }
+  }
+}
+
+provider "prefect" {
+  account_id = "your Account/Organization ID"
+  api_key    = "your API Key"
+}
+```
+
+You can optionally configure your provider through environment variables in the following way, which implicitly injects the same values as the example above. See our [provider documentation](https://registry.terraform.io/providers/PrefectHQ/prefect/latest/docs) for the full attribute schema + related environment variable names.
+
+```shell
+export PREFECT_CLOUD_API_KEY="your API Key"
+export PREFECT_CLOUD_ACCOUNT_ID="your Account/Organization ID"
+```
+
+```terraform
+provider "prefect" {}
+```
+
+The optional `account_id` and `workspace_id` attributes set default values, so that any subsequent resources will inherit those values. Set a `workspace_id` if your use case calls for managing only a single Workspace
```

Comment:
is `account_id` optional?

Reply:
for now, it is. all the account-scoped resources have an account_id override in them as well